.gush.yml

```diff
@@ -1,8 +1,21 @@
-adapter: github
-issue_tracker: github
+# Gush configuration file, any comments will be lost.
 meta-header: "This file is part of Gush package.\n\n(c) 2013-2015 Luis Cordova <cordoval@gmail.com>\n\nThis source file is subject to the MIT license that is bundled\nwith this source code in the file LICENSE."
-pr_type: ['feature', 'bugfix', 'refactor', 'tests', 'remove', 'style', 'documentation', 'security']
+pr_type:
+    - feature
+    - bugfix
+    - refactor
+    - tests
+    - remove
+    - style
+    - documentation
+    - security
 table-pr:
     fixed_tickets: ['Fixed Tickets', '']
-    license: ['License', MIT]
+    license: [License, MIT]
 base: master
+repo_adapter: github
+issue_tracker: github
+repo_org: gushphp
+repo_name: gush
+issue_org: gushphp
```

Comment:
Actually that change would make it inconsistent with the option `--issue-org` :confused: 
The issue-tracker can be part of a repository or a standalone issue-tracker like Jira (which actually uses project-name instead of a repo-name). Even issue_name is not descriptive, name of what?

Taking a quick look at most issue-trackers they seem to prefer the usage of project.
So these option will be named:

* issue_project_org `--issue-org`
* issue_project_namespace `--issue-namespace`.

Some issue-tracker don't even use the organizations, but only projects.
For the future we can separate this using a feature interface (`IssueTrackerWithOrg`) and update the adapters, for now that is to much work and breakage. If the issue-tracker doesn't use it its simple ignored :)